docs/software/integrations/mqtt/mosquitto.mdx

```diff
@@ -47,4 +47,54 @@ sidebar_position: 1
    brew services restart mosquitto
    ```
 
-7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
\ No newline at end of file
+7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
+
+
```

Comment:
Perhaps using tabs for various OS's here would be convenient?

Reply:
think I added them correctly